src/ucp/rndv/proto_rndv.inl

```diff
@@ -242,6 +242,13 @@ ucp_proto_rndv_op_check(const ucp_proto_init_params_t *params,
            (support_ppln || !ucp_proto_rndv_init_params_is_ppln_frag(params));
 }
 
+static UCS_F_ALWAYS_INLINE uct_rkey_t
+ucp_proto_rndv_get_tl_rkey(ucp_request_t *req,
+                           const ucp_proto_multi_lane_priv_t *lpriv)
+{
+    return ucp_proto_common_get_tl_rkey(req->send.rndv.rkey, &lpriv->super);
+}
```

Comment:
why need this wrapper? Imo, `ucp_proto_common_get_tl_rkey` is not really needed also